README.md

```diff
@@ -27,31 +29,109 @@ npm install @dojo/widgets
 ```
 This package contains *all* of the widgets in this repo.
 
-All of the widgets in this repo are on the same release schedule, that is to say, that we release all widgets at the same time. Minor releases may include new widgets and/or features, whereas patch releases may contain fixes to more than 1 widget.
+All of the widgets are on the same release schedule, that is to say, that we release all widgets at the same time.
+Minor releases may include new widgets and/or features, whereas patch releases may contain fixes to more than 1 widget.
 
-In a future release, we will also publish each widget individually to `npm`.
+To use a widget in your application, you will need to import each widget individually:
+
+```ts
+import Button from '@dojo/widgets/button/Button';
+```
+
+Each widget module has a default export of the widget itself, as well as named exports for things such as properties specific to the widget:
+
+```ts
+import Button, { ButtonProperties } from '@dojo/widgets/button/Button';
+```
+
+Because each widget is a separate module, when you create a release build of your application, you will only include the widgets that you have explicitly imported. 
 
 ## Features
 
-Dojo 2 widgets are supported in all evergreen browsers (IE11+, Chrome, Firefox, Safari) as well as popular mobile browsers (Mobile Safari, Chrome on Android).
+- All widgets are supported in all evergreen browsers (IE11+, Chrome, Firefox, Safari) as well as popular mobile browsers (Mobile Safari, Chrome on Android).
+
+- All widgets support accessibility (`a11y`) out of the box.
+
+- All widgets are fully themeable.
+In a future release we will ship with several official Dojo 2 themes that can be applied to the widgets (all themes will live in a separate repo).
+
+- All widgets support internationalisation (`i18n`)
```

Comment:
I believe we're going with American English for consistency in the readmes and docs, so internationalization :)  (I know it was like this before)